src/node.cc

```diff
@@ -909,6 +909,33 @@ Local<Value> WinapiErrnoException(Isolate* isolate,
 #endif
 
 
+static bool IsDomainActive(const Environment* env) {
+  if (!env->using_domains()) {
+    return false;
+  }
+
+  Local<Array> domain_array = env->domain_array().As<Array>();
+  uint32_t domainsArrayLength = domain_array->Length();
```

Comment:
Use snake case for consistency for `domainsArrayLength` -> `domains_array_length`, or just get rid of that temporary, it seems unneeded.